src/Api/Vault/Models/CipherFido2KeyModel.cs

```diff
@@ -70,7 +73,8 @@ public CipherLoginFido2KeyData ToCipherLoginFido2KeyData()
             UserHandle = UserHandle,
             UserDisplayName = UserDisplayName,
             Counter = Counter,
-            Discoverable = Discoverable
+            Discoverable = Discoverable,
+            CreationDate = CreationDate.Value
```

Comment:
issue: it seems we need to use `Value` because `CipherFido2KeyModel.CreationDate` is marked as nullable but `CipherLoginFido2KeyData.CreationDate` is not. I think we need to actually handle the nullable case here, otherwise `.Value` will just initialize with a default value, right?